### PR TITLE
fix: Button overlaps the login container

### PR DESCRIPTION
**Pull request by Valenplayer3000**
This update will only to be merged when it done

> New emoji picker

> fix: Button overleaps the login container

### CODE REVIEW

**Comment by stationaryStation** [comment]
I told you to not change anything else @Valenplayer3000 :/

**Comment by Valenplayer3000** [comment]
# TODO:

- [ ] New Emoji/GIFBox Picker (Almost done)
- [x] Fix conflict with recent commit
- [ ] Allow enabling new Emoji/GIFBox Picker

**Comment by stationaryStation** [comment]
> # TODO:
> 
>     * [ ]  New Emoji/GIFBox Picker (Almost done)
> 
>     * [x]  Fix conflict with recent commit
> 
>     * [ ]  Allow enabling new Emoji/GIFBox Picker

dont

**Comment by Valenplayer3000** [comment]
> I told you to not change anything else @Valenplayer3000 :/

I know but you aren't changing the picker

**Comment by stationaryStation** [comment]
and? it hasn't even been added to the main client yet bc the gifbox api is not complete, plus other things need more priority

**Comment by Valenplayer3000** [comment]
> and? it hasn't even been added to the main client yet bc the gifbox api is not complete, plus other things need more priority

If the gifbox api isn't complete then should I disable the gifbox tab?

File: src/components/ui/messaging/Message/index.tsx [hunk 1]
```diff
@@ -230,7 +231,7 @@ const Message: Component<MessageComponent> = ({
               <Show when={attachment.metadata.type === "Image"}>
                 <img
                   src={`https://autumn.revolt.chat/attachments/${attachment._id}`}
-                  class="w-auto max-h-64"
+                  class="w-auto max-h-64 rounded-lg my-2"
```
Comment (stationaryStation) [inline src/components/ui/messaging/Message/index.tsx:234]: I might need to look how it looks on a dev environment

File: src/components/ui/navigation/Userbar/index.tsx [hunk 2]
```diff
@@ -1,19 +1,38 @@
+// Solid.js
```
Comment (stationaryStation) [inline src/components/ui/navigation/Userbar/index.tsx:1]: No emoji support, yet

File: src/components/ui/settings/index.tsx [hunk 3]
```diff
@@ -384,7 +384,7 @@ const Settings: Component = () => {
       >
         <h2 class="text-center text-xl">Experiments</h2>
         <div class="prose">
-          <h3>Emoji Picker</h3>
+          <h3>Emoji Picker (Experimental)</h3>
```
Comment (stationaryStation) [inline src/components/ui/settings/index.tsx:387]: It's part of the Experiments container, it doesn't need to have the parenthesis

**Review by stationaryStation — COMMENTED** [review]
Wait, at least, until everything is production ready. I am trying to port Solenoid into `revolt-toolkit`, which has a higher priority because it has several QOL features that help development

**Comment by Valenplayer3000** [comment]
@stationaryStation Also I might forgot about putting emoji, this one isn't functional yet

**Comment by stationaryStation** [comment]
@Valenplayer3000 Make sure to at least attach images to your pull request so that I can see the changes without having to pull those from your fork.

File: src/components/ui/messaging/Message/index.tsx (hunk 1 above)
Comment (stationaryStation) [reply src/components/ui/messaging/Message/index.tsx:234]: Hmmm, this looks good

I like how it looks

File: src/components/ui/experiments/Picker/picker.tsx [hunk 4]
```diff
@@ -17,8 +17,8 @@ const [tab, setTab] = createSignal<number>(0);
```
[the remark this answers is not in the document]
Comment (stationaryStation) [reply src/components/ui/experiments/Picker/picker.tsx:17]: Although it looks good, this needs to put on hold for now.

**Comment by Valenplayer3000** [comment]
> @Valenplayer3000 Make sure to at least attach images to your pull request so that I can see the changes without having to pull those from your fork.

Okay 👌🏼

**Comment by stationaryStation** [comment]
Also, for the emoji picker, I suggest to separate it into a different pr, It's missing functionality (Such as listing server emotes) or add an issue to track the progress more easily

**Comment by stationaryStation** [comment]
Tracking it here doesn't make a lot of sense to me

**Comment by Valenplayer3000** [comment]
Ima show you late (because github won't let send the image because of bug with firefox)

**Comment by stationaryStation** [comment]
Can't you just click the "Attach files by..." thing at the bottom of the comment box to attach it?

**Comment by stationaryStation** [comment]
wtf why

**Pull request closed without merging**